doc/integrations/cortx-docker/README.md

```diff
@@ -0,0 +1,95 @@
+    aws configure set aws_access_key_id your_access_key_id --profile default
+    aws configure set aws_secret_access_key your_access_key --profile default
+    aws configure set region us-east-1 --profile default
+
+Step 3: Install goofys.
+
+    mkdir -p $HOME/go/src/github.com/kahing/
+    cd $HOME/go/src/github.com/kahing/
+    git clone https://github.com/kahing/goofys.git
+    export GOPATH=$HOME/go
+    export GOOFYS_HOME=$HOME/go/src/github.com/kahing/goofys/
+    cd $HOME/go/src/github.com/kahing/goofys
+    git submodule init
+    git submodule update
+    PATH=$PATH:$HOME/go/bin; export PATH
+    sudo go install $HOME/go/src/github.com/kahing/goofys
+
+Step 4: Mount Cortx S3 bucket with goofys and start Docker Registry with the bucket as storage. Replace `http://192.168.1.91` with your own S3 endpoint url.
+
+    sudo mkdir /mnt/registry
+    goofys --endpoint http://192.168.1.91 testbucket /mnt/registry
+    docker run -d \
+    -p 5000:5000 \
+    --restart=always \
+    --name registry \
+    -v /mnt/registry:/var/lib/registry \
+    registry:2
+
+###  Test the system 
+
+Step 1: From Docker development machine, build a test docker image `hello` and push it to Docker Registry server. Replace `192.168.1.115` with your own Docker Registry server ip. If encounter "http: server gave HTTP response to HTTPS client" error, add your Docker Registry server ip to docker development machine's daemon configuration json file under "insecure-registries" first. 
+
+Alternatively, if you test Docker development client on same machine as Docker Registry server, replace `192.168.1.115` with `localhost`.
+
+    mkdir hello
+    echo -e "# syntax=docker/dockerfile:1\nFROM scratch\nADD hello /\n" > Dockerfile
+    docker build --tag hello .
+    docker tag hello 192.168.1.115:5000/hello 
+    docker push 192.168.1.115:5000/hello 
+
+Step 2: Check Cortx S3 bucket is now updated with the new docker image. You can see `hello/` docker image is added to the bucket's `repositories` folder. Replace `http://192.168.1.91` with your own S3 endpoint url.
+
+    aws s3 --endpoint "http://192.168.1.91" ls s3://testbucket/docker/registry/v2/repositories/
+
+Step 3: From Docker deployment machine, pull docker image from Docker Registry server. Then you can start to use the downloaded docker image as usual.
+
+    C:\tmp>docker pull 192.168.1.115:5000/hello                                                                             
+    Using default tag: latest                                                                                               
+    latest: Pulling from hello                                                                                              
+    2db29710123e: Pull complete                                                                                             
+    Digest: sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4                                         
```

Comment:
@wb-droid, can you remove or hide the digest

Reply:
@mukul-seagate11, sure, it's removed.